joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

wheel_front_left_controller:
  type: effort_controllers/JointVelocityController
  joint: joint_wheel_front_left
  pid: {p: 100.0, i: 0.01, d: 1.0}

wheel_front_right_controller:
  type: effort_controllers/JointVelocityController
  joint: joint_wheel_front_right
  pid: {p: 100.0, i: 0.01, d: 1.0}

wheel_back_left_controller:
  type: effort_controllers/JointVelocityController
  joint: joint_wheel_back_left
  pid: {p: 100.0, i: 0.01, d: 1.0}

wheel_back_right_controller:
  type: effort_controllers/JointVelocityController
  joint: joint_wheel_back_right
  pid: {p: 100.0, i: 0.01, d: 1.0}